src/components/modals/ImportAccounts/steps/03-step-import.js

```diff
@@ -0,0 +1,168 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import type { Account } from '@ledgerhq/live-common/lib/types'
+
+import { getBridgeForCurrency } from 'bridge'
+
+import Box from 'components/base/Box'
+import Button from 'components/base/Button'
+import Spinner from 'components/base/Spinner'
+import IconExchange from 'icons/Exchange'
+
+import AccountRow from '../AccountRow'
+
+import type { StepProps } from '../index'
+
+class StepImport extends PureComponent<StepProps> {
+  componentDidMount() {
+    this.startScanAccountsDevice()
+  }
+
+  componentWillUnmount() {
+    if (this.scanSubscription) {
+      this.scanSubscription.unsubscribe()
+    }
+  }
+
+  scanSubscription = null
+
+  startScanAccountsDevice() {
+    const { currency, currentDevice, setState } = this.props
+    try {
+      if (!currency) {
+        throw new Error('No currency to scan')
+      }
+
+      if (!currentDevice) {
+        throw new Error('No device')
+      }
```

Comment:
these 2 cases shouldn't happen, meaning we should probably have the props type to be directly currency: Currency, etc.. because you don't want to render until these satisfies. 

had same issue on the Send flow, current solution is not so satisfying but still better in types I feel https://github.com/LedgerHQ/ledger-live-desktop/blob/master/src/components/modals/Send/03-step-verification.js#L45

anyway, it's for later